.babelrc

```diff
@@ -1,7 +1,6 @@
 {
-  "presets": [ "react", "es2015" ],
-  "plugins": [
-    "transform-object-assign",
```

Comment:
`preset: env` is the replacement for `preset: es2015` (more details [here](http://babeljs.io/env)), and Electron (because Chromium) natively supports things like spread syntax, object assign and async/await - so we can drop lots of the customizations in here.